### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Pailong123** [qbcore-framework/qb-weathersync#71]
I have a QB Core framework, and has this qb-weathersync, but I also have vMenu. I am confused because the server has flicking day and night, I think the weathers on both resources are overriding each other. what to do to fix this?

**Comment by TheRealToxicDev** [comment]
> I have a QB Core framework, and has this qb-weathersync, but I also have vMenu. I am confused because the server is flicking day and night, I think the weathers on both resources are overriding each other. what to do to fix this?

Correct, vMenu automatically cycles time every 15 minutes which is probably interfering with QBCore's weather sync..

To fix this vMenu has options in your `permissions.cfg` to disable its weather sync this is what i did and fixed the issue for me 👌🏻  hope this helps

Change these both to `false`: ![Example](https://img.toxicdev.me/d26ba6.png)

**Comment by nzkfc** [comment]
You should prob get rid of vMenu, I assume you came from a server running vMenu for character creation/saving, common from older ESX servers, but once you get used too multicharacter, illenium-appearance and admin menus it really removes the need for vMenu, I was in the same boat 2 years ago.
